engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/IrPersistance.java

```diff
@@ -70,6 +70,8 @@
 @Persistable(clazz = Warning.TypeMismatch.class, id = 792)
 @Persistable(clazz = Warning.NoSuchMethod.class, id = 793)
 @Persistable(clazz = Warning.NonUnitTypeUsedOnValueLevel.class, id = 794)
+@Persistable(clazz = Warning.UnusedImport.class, id = 795)
```

Comment:
- surprising
- do we want warnings to be persisted in caches?
- isn't it better to fail the PR integration and require fixes to be made?

Reply:
It is. Persistance for UnusedImports warning IR removed in e3c66dc25aef8be7142564535e658260b365bba0. This was just some left over.